File: src/main/java/com/vjiki/music/service/ShortService.java
package com.vjiki.music.service;

import java.util.List;
import java.util.UUID;

import com.vjiki.music.dto.ShortResponse;
import com.vjiki.music.pagination.CursorPageResponse;

public interface ShortService {

    List<ShortResponse> getShorts(UUID userId);

    CursorPageResponse<ShortResponse> getShortsPage(UUID userId, int limit, String cursor);
}

File: src/main/kotlin/com/vjiki/music/service/ShortService.kt
package com.vjiki.music.service

import com.vjiki.music.dto.ShortResponse
import com.vjiki.music.pagination.CursorPageResponse
import java.util.UUID

interface ShortService {
    fun getShorts(userId: UUID): List<ShortResponse>

    fun getShortsPage(userId: UUID, limit: Int, cursor: String?): CursorPageResponse<ShortResponse>
}

